src/ShapeDiver/PlatformBackendUtils.ts

```diff
@@ -277,4 +296,135 @@ export const queryUserCreditUsage = async (sdk: SdPlatformSdk, user_id: string,
   return items;
 }
 
+/**
+ * Decides which filter to use.
+ */
+export enum NotifyUsersOrganizationFilter 
+{
+  /**
+   * If no filter is to be applied.
+   */
+  NoFilter,
+  /**
+   * Check for users that are in organization.
+   */
+  InOrganization,
+
+  /**
+   * Check for users which are not in organization.
+   */
+  NotInOrganization,
+}
+
+export interface NotifyUsersUserOptions 
+{
+  subscribed_plan_name: string;
+  features_of_user?: string | Array<string>;
+  organization_filter?: NotifyUsersOrganizationFilter;
+  organization_roles?: string | Array<string>;
+  dry_run?: boolean;
+
+  /**
+   * If not full result of users is returned ( if 'offset' exists in response), pass offset to get next batch of results.
+   */
+  offset?: string;
+}
+
+/**
+ * The notification options parameters.
+ */
+export interface NotifyUsersNotificationOptions 
+{
+  type: SdPlatformNotificationType;
+  href?: string;
+  description: string;
+}
+
+
+export const notifyUsers = async (sdk: SdPlatformSdk, notify_users_user_options: NotifyUsersUserOptions, notification_options: NotifyUsersNotificationOptions): Promise<SdPlatformQueryResponse<SdPlatformResponseUserMinimal>> => 
+{
+  const uo = notify_users_user_options;
+
+  const filter = {
+    "chargebee_user.type": "plan",
+    "chargebee_user.data->name": uo.subscribed_plan_name
+  };
+
+  // Fatures of user filter
+  if (Array.isArray(uo.features_of_user))
+  {
+    for (let feature of uo.features_of_user)
+    {
+      filter[`user_features->feature->${feature.trim()}`] = true;
+    }
+  }
+  else if (uo.features_of_user && uo.features_of_user != "")
+  {
+    filter[`user_features->feature->${uo.features_of_user}`] = true;
+  }
+
+  // organization filter.
+  if (uo.organization_filter == NotifyUsersOrganizationFilter.InOrganization)
+  {
+    filter["organization_id[!?]"] = null;
+  }
+  else if (uo.organization_filter == NotifyUsersOrganizationFilter.NotInOrganization)
+  {
+    filter["organization_id[?]"] = null;
+  }
+
+  // users
+  if (Array.isArray(uo.organization_roles))
+  {
+    filter["organization_role[,]"] = uo.organization_roles.map(x => x.trim());
+  }
+  else if (uo.organization_roles && uo.organization_roles != "")
+  {
+    filter["organization_role"] = uo.organization_roles;
+  }
+
+  console.log(filter);
+
+  try
+  {
+    var users = await sdk.users.query({
+      filters: filter,
+      limit: 100,
+      strict_limit: true
+    });
+  } catch (ex)
+  {
+    console.log(ex);
+    throw ex;
+  }
```

Comment:
@luka712 chunks of 100 users are fine, but we need to iterate through all which we find. Ideally the `console.table` output should happen for each batch, maybe let's reduce batch size to 25